### PR TITLE
Add job periodic-prow-image-autobump

### PR DESCRIPTION
**Pull request by hongkailiu**
https://jira.coreos.com/browse/DPTP-518

### CODE REVIEW

**Comment by hongkailiu** [comment]
/hold
dep. on https://github.com/openshift/ci-tools/pull/91

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -271,6 +271,43 @@ periodics:
...
+      - /usr/bin/autobump.sh
+      image: registry.svc.ci.openshift.org/ci/autobumper:latest
+      imagePullPolicy: Always
+      name: "autobump"
+      resources:
+          requests:
+            cpu: 500m
+      env:
+      ### TODO: update those tbu(s)
+      - name: github_login
```
Comment (stevekuznetsov) [inline ci-operator/jobs/infra-periodics.yaml]: We can hard-code this to be `openshift-merge-robot` (and in general should not need it)

File: ci-operator/jobs/infra-periodics.yaml [hunk 2]
```diff
@@ -271,6 +271,43 @@ periodics:
...
+        value: "tbu"
+      - name: github_token
```
Comment (stevekuznetsov) [inline ci-operator/jobs/infra-periodics.yaml]: Why not pass as a flag?

File: ci-operator/jobs/infra-periodics.yaml (hunk 1 above)
Comment (hongkailiu) [reply ci-operator/jobs/infra-periodics.yaml]: Removed.

File: ci-operator/jobs/infra-periodics.yaml (hunk 2 above)
Comment (hongkailiu) [reply ci-operator/jobs/infra-periodics.yaml]: Done.

**Pull request merged**